packages/conf-numa/conf-numa.0.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Steve Bleazard <stevebleazard@googlemail.com>"
+authors: "Steve Bleazard <stevebleazard@googlemail.com>"
+homepage: "https://www.github.com/stevebleazard/ocaml-conf-numa"
+bug-reports: "https://www.github.com/stevebleazard/ocaml-conf-numa/issues"
+license: "MIT"
+dev-repo: "https://www.github.com/stevebleazard/ocaml-conf-numa.git"
+
+build: [
+  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include files/test.c"]
+]
+
+depexts: [
+  [["debian"] ["libnuma-dev"]]
+  [["ubuntu"] ["libnuma-dev"]]
+  [["centos"] ["numactl-libs" "numactl-devel"]]
+  [["fedora"] ["numactl-libs" "numactl-devel"]]
+  [["opensuse"] ["libnuma-devel"]]
+  [["alpine"] ["libnuma-dev"]]
+]
+
+available: [ ocaml-version >= "4.03.0" & os != "darwin" & os != "freebsd" & os != "openbsd" ]
```

Comment:
Why do you have a >=4.03 requirement here?

Reply:
This got carried over from the split off from numalib, removed